synapse/lib/stormtypes.py

```diff
@@ -6228,6 +6228,10 @@ class Layer(Prim):
                       {'name': 'url', 'type': 'str', 'desc': 'A telepath URL of the target layer/feed.', },
                       {'name': 'offs', 'type': 'int', 'desc': 'The local layer offset to begin pushing from',
                        'default': 0, },
+                      {'name': 'queue_size', 'type': 'int', 'desc': 'The queue size of the pusher.',
+                       'default': 10_000},
+                      {'name': 'chunk_size', 'type': 'int', 'desc': 'The chunk size of the pusher.',
```

Comment:
Might be good to add a little more context to what "chunk size" means here, maybe something like "The chunk size to use when pushing batches of edits."?

Reply:
Updated.